src/systems/explicit_system.C

```diff
@@ -45,6 +46,8 @@ ExplicitSystem::~ExplicitSystem ()
 {
   // clear data
   this->clear();
+
+  remove_vector("RHS Vector");
```

Comment:
I see. Thanks. We then can completely remove the ExplicitSystem destructor, right?